io_scene_nif/__init__.py

```diff
@@ -103,6 +103,7 @@ def menu_func_import(self, context):
 # noinspection PyUnusedLocal
 def menu_func_export(self, context):
     self.layout.operator(operators.nif_export_op.NifExportOperator.bl_idname, text="NetImmerse/Gamebryo (.nif)")
+    # self.layout.operator(operators.kf_export_op.KfExportOperator.bl_idname, text="NetImmerse/Gamebryo (.kf)")
```

Comment:
I wasn't really sure if a separate Kf export operator makes sense. I had coded it and then realized that the Nif operator's output selector (all/xnif/geometry only/anims only) is probably more useful.

Reply:
I will move a lot of the kf code out to the animation export module. If we then decide to split kf export into its own operator, that will be really easy. I also need to split up the export_keyframes function and organize all its components nicely to avoid building another monolithic monster there... 🤣

Reply:
And that is how I got to have so many abandoned branches. 😆